baseplate/metrics.py

```diff
@@ -85,22 +85,62 @@ def send(self, serialized_metric):
 
 class BufferedTransport(object):
     """A transport which wraps another transport and buffers before sending."""
+    MAX_MESSAGE_SIZE = 65507
+
     def __init__(self, transport):
         self.transport = transport
         self.buffer = []
 
     def send(self, serialized_metric):
-        self.buffer.append(serialized_metric)
+        if len(serialized_metric) >= self.MAX_MESSAGE_SIZE:
```

Comment:
Should this be `>` like it is in `make_messages`?

Reply:
The idea being that a single metric can be `MAX_MESSAGE_SIZE` since there won't be a newline character added (I think?).

Reply:
yep